dags/vz_cr3_extract_ocr_narrative.py

```diff
@@ -0,0 +1,70 @@
+import os
+from pendulum import datetime, duration
+
+from airflow.decorators import task
+from airflow.models import DAG
+from airflow.operators.docker_operator import DockerOperator
+
+from utils.slack_operator import task_fail_slack_alert
+
+DEPLOYMENT_ENVIRONMENT = os.getenv("ENVIRONMENT", "development")
+
+default_args = {
+    "owner": "airflow",
+    "description": "Extracts the diagram and narrative out of CR3 pdfs",
+    "depends_on_past": False,
+    "start_date": pendulum.datetime(2019, 1, 1, tz="America/Chicago"),
+    "email_on_failure": False,
+    "email_on_retry": False,
```

Comment:
We don't use email alerts in airflow V2, I think these can go.

Reply:
We don't use them, but do they default as True? 
https://airflow.apache.org/docs/apache-airflow/1.10.6/_api/airflow/models/index.html#airflow.models.BaseOperator

Reply:
Chia, please forgive me! I absolutely didn't notice that you were setting them to false which is 100% correct. I saw that they were being set, and didn't read closely enough. This is perfect. Thank you!